internal/builders/golang/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package golang provides a Builder implementation for golang.
+package golang
```

Comment:
What is this used for?
Go builder specific docs rendered by godoc?
I ask because it doesn't seem to be used.

Reply:
for now, the build pipeline imports this just so `init` is called, later it will be a plugin and things would be different :P